chat/src/main/java/com/github/twitch4j/chat/TwitchChatBuilder.java

```diff
@@ -195,6 +196,21 @@
     @With
     private boolean enableMembershipEvents = true;
 
+    /**
+     * Whether join failures should result in removal from current channels.
+     *
+     * @see ChannelRemovedPostJoinFailureEvent
+     */
+    @With
+    private boolean removeChannelOnJoinFailure = false;
```

Comment:
is this too verbose for a variable name

Reply:
I think the name is fine because it's obvious what it's doing at a glance.